### PR TITLE
cloud: Add release notes

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,55 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    To access Data Service, navigate to the [**Data Service**](https://tidbcloud.com/console/dataservice) page in the left navigation pane. For more information, see the following documentation:
+
+    - [Data Service Overview](/tidb-cloud/data-service-overview.md)
+    - [Get Started with Data Service](/tidb-cloud/data-service-get-started.md)
+    - [Get Started with Chat2Query API](/tidb-cloud/use-chat2query-api.md)
+
+- Support decreasing the size of TiDB, TiKV, and TiFlash nodes to scale in a [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) cluster.
+
+    You can decrease the node size [via the TiDB Cloud console](//tidb-cloud/scale-tidb-cluster.md#change-node-size) or [via the TiDB Cloud API (beta)](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/UpdateCluster).
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:34]:
```suggestion
    You can decrease the node size [via the TiDB Cloud console](/tidb-cloud/scale-tidb-cluster.md#change-node-size) or [via the TiDB Cloud API (beta)](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/UpdateCluster).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,55 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+- Support a new GCP region for the [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) feature of [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters: `Tokyo (asia-northeast1)`.
+
+    The feature can help you to migrate data from MySQL-compatible databases in Google Cloud Platform (GCP) to your TiDB cluster easily and efficiently.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:38]:
```suggestion
    The feature can help you migrate data from MySQL-compatible databases in Google Cloud Platform (GCP) to your TiDB cluster easily and efficiently.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 3]
```diff
@@ -8,6 +8,55 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    For more information, see [TiDB Cloud cluster events](tidb-cloud/tidb-cloud-events.md).
+
+- Add the **Database Status** tab to the **Monitoring** page for [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters, which displays the following database-level metrics:
+
+    - QPS Per DB
+    - Average Query Duration Per DB
+    - Failed Queries Per DB
+
+  With these metrics, you can monitor the performance of individual databases, make data-driven decisions, and take actions that improve the performance of your applications.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:56]:
```suggestion
  With these metrics, you can monitor the performance of individual databases, make data-driven decisions, and take actions to improve the performance of your applications.
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 4]
```diff
@@ -8,6 +8,55 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    For more information, see [Migrate MySQL-compatible databases to TiDB Cloud using Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md).
+
+**Console changes**
+
+- Introduce the **Events** page for [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) clusters, which provides the records of main changes to your cluster.
+
+    On this page, you can view the event history for the last 7 days and track important details such as the trigger time and the user who initiated an action. For example, you can view events such as when a cluster was paused or who modified the cluster size.
+
+    For more information, see [TiDB Cloud cluster events](tidb-cloud/tidb-cloud-events.md).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:48]:
```suggestion
    For more information, see [TiDB Cloud cluster events](/tidb-cloud/tidb-cloud-events.md).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 5]
```diff
@@ -8,6 +8,55 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    In addition, TiDB Cloud provides the [Chat2Query API](/tidb-cloud/use-chat2query-api.md), a RESTful interface that allows you to generate and execute SQL statements using AI.
+
+    To access Data Service, navigate to the [**Data Service**](https://tidbcloud.com/console/dataservice) page in the left navigation pane. For more information, see the following documentation:
+
+    - [Data Service Overview](/tidb-cloud/data-service-overview.md)
+    - [Get Started with Data Service](/tidb-cloud/data-service-get-started.md)
+    - [Get Started with Chat2Query API](/tidb-cloud/use-chat2query-api.md)
+
+- Support decreasing the size of TiDB, TiKV, and TiFlash nodes to scale in a [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) cluster that is hosted on AWS.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:32]:
```suggestion
- Support decreasing the size of TiDB, TiKV, and TiFlash nodes to scale in a [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier) cluster that is hosted on AWS and created after December 31, 2022.
```

**Pull request merged**